Makefile

```diff
@@ -43,25 +52,26 @@ doc:
 
 .PHONY: release
 release:
-	$(SED) -i 's/\"version\":\ \"[0-9]\+\.[0-9]\+\.[0-9]\+\"/\"version\":\ \"$(VERSION)\"/' package.json
-	$(SED)  -i "s/Unreleased/`date +%Y-%m-%d`/" CHANGELOG.md
+	$(SED) -i '' 's/\"version\":\ \"[0-9]\+\.[0-9]\+\.[0-9]\+\"/\"version\":\ \"$(VERSION)\"/' package.json
+	$(SED) -i '' 's/Unreleased/`date +%Y-%m-%d`/' CHANGELOG.md
```

Comment:
I don't understand the point of this change (and the similar ones below). It looks wrong to me and the documentation you linked to doesn't shed any light.

Perhaps you didn't set the VERSION when you called `make release`?

It should be called like this: `make release VERSION=1.3`

Reply:
Excuse me @jcbrand, it seems I linked a wrong reference. [Here is the right one.](https://stackoverflow.com/a/2321958/2980087)
In MacOS the parameter `-i` is looking for an extension to create a backup before changes.

I hope my changes work in macOS and Linux both.

Reply:
I tested my changes on Ubuntu. Unfortunately it didn't work.
```
$ sed -i '' 's/hel/wel/' hello.c
sed: s/hel/wel/: No such file or directory
```

Please advise how should we support both platforms in this case?

Reply:
On Mac GNU sed can still be used.

`brew install gnu-sed`
`SED=gsed make release VERSION=1.3`

Perhaps you'd be wiling to mention this in the Makefile (in the `help` text)?

Reply:
done